bears/natural_language/WriteGoodLintBear.py

```diff
@@ -22,6 +23,14 @@ class WriteGoodLintBear:
     CAN_DETECT = {'Formatting', 'Grammar'}
 
     @staticmethod
+    @deprecate_settings(allow_passive_voice='check_passive_voice',
+                        allow_so_beginning='check_so_beginning',
+                        allow_adverbs='check_adverbs',
+                        allow_repeated_words='check_repeated_words',
+                        allow_there_is='check_there_is',
+                        allow_ambiguous_words='check_ambiguos_words',
+                        allow_extra_words='check_extra_words',
+                        allow_cliche_phrases='check_cliche_exists')
```

Comment:
don't all of those need inverts? Btw. I think you'll want to make an invert function somewhere and import it instead of doing a lambda every time?